joplin/static/js/editor.js

```diff
@@ -0,0 +1,27 @@
+$(function() {
+
+  $('.js-proxy-click').click(function() {
+    let $this = $(this);
+    $this.text($this.data('clicked-text'));
+
+    let $button;
+
+    let proxyByName = $this.data('proxyByName');
+    if (proxyByName) {
+      $button = $(`[name="${proxyByName}"]`);
+    }
+
+    let proxyByClass = $this.data('proxyByClass');
+    if (proxyByClass) {
+      $button = $(`.${proxyByClass}`);
+    }
+
+    if (!$button) {
+      console.error(`Data attributes: ${$this.data()}`)
+      throw new Error('Unable to find a button. Did you specify data-proxy-by-name or data-proxy-by-class?');
+    }
+
+    $button.click();
```

Comment:
Why did we have to do this proxy click dance instead of letting the new buttons do the submit actions? Guessing it has to do with limitations to the existing html layouts? This seems a little hacky, but I'm not above it.

Reply:
I thought it might've saved us some headaches if they were doing anything weird in a JS click handler, but it's probably easier for future readers if I just submit the form. I can give that a whirl and see where it lands.

Reply:
I have them wired up to hit submit in my PR: https://github.com/cityofaustin/joplin/pull/132

Reply:
OK Sweet, so it seems like this PR isn't even necessary? I was just trying to wire something up quickly to kinda see things work, but @briaguya has the real deal going on.